src/goMain.ts

```diff
@@ -83,6 +83,7 @@ export function activate(ctx: vscode.ExtensionContext): void {
 			});
 		}
 		ctx.globalState.update('goroot', currentGoroot);
+		TempFileProvider.registerStore(ctx.globalState);
```

Comment:
Move this to the very top. Just to make sure that the globalState is available in the TempFileProvider right from the start